admin-dev/themes/new-theme/js/app/utils/datepicker.js

```diff
@@ -65,27 +67,28 @@ const replaceDatePicker = () => {
 const init = function initDatePickers() {
   const $datePickers = $('.datepicker input[type="text"]');
   $.each($datePickers, (i, picker) => {
-    $(picker).datetimepicker(
-      {
+    $(picker)
+      .datetimepicker({
         locale: window.full_language_code,
-        format: $(picker).data('format') ? $(picker).data('format') : 'YYYY-MM-DD',
+        format: $(picker).data('format')
+          ? $(picker).data('format')
+          : 'YYYY-MM-DD',
         sideBySide: true,
         icons: {
           time: 'time',
           date: 'date',
           up: 'up',
           down: 'down',
         },
-      },
-    ).on(
-      'dp.show',
-      replaceDatePicker,
-    ).on(
-      'dp.hide',
-      () => {
+      })
+      .on('dp.show', replaceDatePicker)
+      .on('dp.hide', () => {
         $(window).off('resize', replaceDatePicker);
-      },
-    );
+      })
+      .on('dp.change', (e) => {
```

Comment:
Just a question,  I wonder if this is not overkill Could you not simply listen for the `dp.change` event like here:
https://github.com/PrestaShop/PrestaShop/blob/8e7b9eafd46cf7535d641928e867be147c6119c6/admin-dev/themes/new-theme/js/pages/product/edit/product-partial-updater.js#L64

I admit that since this is in an iframe it might be more complicated

Reply:
I tried, looks like it doesn't work at all